base/logger/R/logifnot.R

```diff
@@ -1,14 +1,15 @@
 #' Logger message if conditions are not met
 #'
-#' Similar to [base::stopifnot], but allows you to use a custom message and 
+#' Similar to [base::stopifnot], but allows you to use a custom message and
 #' logger level. If all conditions are `TRUE`, silently exit.
 #'
-#' Conditions can be vectorized, or can return non-logical values.The 
-#' underlying function automatically applies `isTRUE(all(.))` to the 
+#' Conditions can be vectorized, or can return non-logical values.The
+#' underlying function automatically applies `isTRUE(all(.))` to the
 #' conditions.
 #'
 #' @param msg Logger message to write, as a single character string.
-#' @param ... Conditions to evaluate 
+#' @param ... Conditions to evaluate
+#' @param ... other arguments passed on to severeifnot
```

Comment:
This line shouldn't be needed and I wasn't thinking clearly when I suggested it! 🤦 
 I think the actual change needed is to not have Roxygen document `check_conditions` at all -- see my other suggestions below.
 
```suggestion
```